src/find-pypy.ts

```diff
@@ -93,10 +93,17 @@ function parsePyPyVersion(versionSpec: string): IPyPyVersionSpec {
   const versions = versionSpec.split('-').filter(item => !!item);
 
   if (versions.length < 2) {
-    throw new Error(
-      "Invalid 'version' property for PyPy. PyPy version should be specified as 'pypy-<python-version>'. See readme for more examples."
+    core.setFailed(
+      "Invalid 'version' property for PyPy. PyPy version should be specified as 'pypy-<python-version>'. See README for examples and documentation."
     );
   }
+
+  if (!validatePythonVersion(versions[1])) {
+    core.setFailed(
+      "Invalid 'version' property for PyPy. PyPy version should be specified as 'pypy-<python-version>'. See README for examples and documentation."
```

Comment:
PyPy version should be specified as 'pypy-<python-version>' -> 
```
Both Python version and PyPy versions should satisfy SemVer notation.
```

Reply:
Done.